### DISCUSSION TITLE
Getting Mailbox global forwarder and rules.

### DISCUSSION THREAD

**Discussion by darshannnn** [ecederstrand/exchangelib#1174] [Q&A]
Hi there,
I'm trying to read mailbox settings to get if there are any global mailbox forwarders set and also get the mailbox rules.
How can I do that?
Thanks

**Comment by ecederstrand** [comment]
Inbox rules are not yet supported. See https://github.com/ecederstrand/exchangelib/issues/233

**Reply by darshannnn** [reply]
Thanks, is there a way to make generic API requests using your library?
I was thinking may be I can do that as a work around.

**Reply by ecederstrand** [reply]
Yes. The easiest is probably to start with a simple service, e.g. https://github.com/ecederstrand/exchangelib/blob/master/exchangelib/services/get_room_lists.py and then follow the code from `call()`. There are various methods in the `EWSService` class you can hook into, depending on whether you want to send lxml objects or text to EWS, and whether you want text or lxml objects in return.
